invisible_cities/cities/esmeralda.py

```diff
@@ -429,37 +190,16 @@ def esmeralda(files_in, file_out, compression, event_range, print_mod,
 
     threshold_and_correct_hits_high = fl.map(hits_threshold_and_corrector(threshold_charge=cor_hits_params['threshold_charge_high'], **cor_hits_params_),
                                              args = 'hits',
-                                             out  = 'cor_high_th_hits')
+                                             out  = 'hits')
```

Comment:
you can use 
`item = 'hits'`

Reply:
Good to know, I didn't know about that option.